backend/data-access/instructorsData.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
+const PATH = './data/instructors.json';
+
+/**
+ * Get all instructors from database
+ * 
+ * @returns {Array} a list of all instructors stored in BD
+ */
+const getInstructors = () => {
+    return JSON.parse(fs.readFileSync(PATH));
+}
+
+/**
+ * Get an instructor with given id from database
+ * 
+ * @param {string} id 
+ * 
+ * @returns {object | null} instructor with given id, null if not found
```

Comment:
Why `null` and not just return the `undefined` that the `Array.find` returns?
This way you can just do a straight `return instructors.find(instructor => instructor.id.$oid === id);`